### DISCUSSION TITLE
Frage zu Fallback Option

### DISCUSSION THREAD

**Discussion by JauntyJosef** [wiggal/GEN24_Ladesteuerung#60] [Q&A]
In der config.ini gibt es ja die Fallback Option, wo man 0, 1 oder 2 auswählen kann.
Was bedeuten die Zahlen? 0 wird wahrscheinlich für "aus" stehen, aber 1 und 2?

**Comment by wiggal** [comment]
Hallo JauntyJosef 
hier die Erklärung aus der config.ini.info:

[Fallback]
; Fallback_on = 0 Programmfunktion ganz ausgeschaltet, KEINE Prüfung ob eine Fallbackzeit geschrieben ist
; Fallback_on = 1 ausgeschaltet, Prüfung ob eine Fallbackzeit geschrieben ist, evtl. auf 0 setzen
; Fallback_on = 2 eingeschaltet
; War Fallback bereits aktiv erst mit Fallback_on = 1 Fallbackzeit auf 0 setzen lassen und dann auf Fallback_on = 0 gehen
Fallback_on = 0

das bedeuten in meinen Skripten ist die Fallbackfunktion  mit 0 aus, bzw. 2  ein.
 
War die  Fallbackfunktion eingeschaltet, wird die Fallbackzeit am Wechselrichter geschrieben.
Würde man nun die Fallbackfunktion mit 2  im Skript einfach abschalten, würde der Wechselrichter nach Ablauf der Fallbackzeit trotzdem in den Fallback laufen.
Um den Fallback am Wechselrichter auszuschalten muss man als Fallbackzeit 0 setzen, dies erreicht man durch Fallback_on = 1.

Ich hoffe ich hab's einigermaßen verständlich beschrieben.

Schönen Gruß

**Reply by JauntyJosef** [reply]
Vielen lieben Dank! Wer lesen kann, ist klar im Vorteil. :-)

Was bewirkt diese Fallback Funktion eigentlich genau? Werden die Modbus Register auf "Werkseinstellung" zurück gesetzt, oder wie kann man sich das vorstellen?

**Reply by wiggal** [reply]
ja, genau die Register werden zurück gesetzt,
is der selbe Effekt wie das ab- und wieder anschalten der Modbussteuerung in der WR-Oberfläche

**Reply by JauntyJosef** [reply]
Ah, alles klar. Vielen Dank!!

Jetzt muss ich nur noch schauen, wo ich das Passwort für das editieren der config in der Web Oberfläche finde.

**Reply by wiggal** [reply]
html/config.php 😉

**Comment by JauntyJosef** [comment]
Gefunden! Danke sehr!
